components/Templates/includes/functions-view_template.php

```diff
@@ -116,9 +124,58 @@ function frontier_if_block( $atts, $code ) {
 				break;
 		}
 	} else {
-		$field_data = $pod->field( $atts['field'] );
 
-		$field_type = $pod->fields( $atts['field'], 'type' );
+		/**
```

Comment:
Were the changes at https://github.com/pods-framework/pods/pull/5471/files#diff-3af4ca77fc3cd86765f464bb6fa7000725503257e80ccc885464b3ad43ab36c4L536-L540 not enough to do what you wanted here, is this part of the code still necessary? It seems that the previous field() alone would do what this does

Reply:
No, currently only using the field method isn't enough. The field type and Pod object needs to be updated as well so traversals will work properly.
However, I do agree that this might not be the best way to do this.